src/OmniSharp.Host/Internal/CommandOptionExtensions.cs

```diff
@@ -14,5 +14,10 @@ public static T GetValueOrDefault<T>(this CommandOption opt, T defaultValue)
 
             return defaultValue;
         }
+
+        public static bool GetValueOrDefault(this CommandOption opt, bool defaultValue)
+        {
+            return opt.Value() == "on" || defaultValue;
```

Comment:
Should this be a case-insensitive comparison?

Reply:
The command line stuff specifically uses lowercase, but I'll change in case that ever changes.